models/Translation/Listing/Dao.php

```diff
@@ -73,11 +73,10 @@ public function getCount()
      */
     public function getAllTranslations()
     {
-        $cacheKey = $this->getDatabaseTableName().'_data';
-        if (!$translations = Cache::load($cacheKey)) {
+        $queryBuilder = $this->getQueryBuilder(['*']);
+        $cacheKey = $this->getDatabaseTableName().'_data_' . md5((string)$queryBuilder);
+        if (!empty($this->model->getConditionParams()) || !$translations = Cache::load($cacheKey)) {
```

Comment:
if `addConditionParam()` called on listing, then it will overwrite the cache everytime... Is this expected behavior?

Reply:
either we need to include all the condition params to the cache key, or we deactivate caching as soon as condition params are set. 
I was thinking that adding the condition params to the cache will create tons of cache keys, that are not used very often anyway (because the queries are very specific...)

Reply:
but it will not deactivate caching completely as on line https://github.com/pimcore/pimcore/pull/8427/files#diff-f28aba119c2334883e102db25bad594b03bb7900a84da7e69f470a4a3a40913aR101 still creates a new cache item with generated cache key..

Reply:
good catch!

Reply:
please recheck